presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -19,6 +19,26 @@ Presto Geospatial functions support the Well-Known Text (WKT) form of spatial ob
 * ``MULTIPOLYGON (((0 0, 4 0, 4 4, 0 4, 0 0), (1 1, 2 1, 2 2, 1 2, 1 1)), ((-1 -1, -1 -2, -2 -2, -2 -1, -1 -1)))``
 * ``GEOMETRYCOLLECTION (POINT(2 3), LINESTRING (2 3, 3 4))``
 
+Use ST_GeometryFromText and ST_GeomFromBinary functions to create geometry objects from WKT or WKB.
+
+The SphericalGeography type provides native support for spatial features represented on "geographic" coordinates (sometimes called "geodetic" coordinates,
```

Comment:
I think this would look better with italics (surround with asterisks) rather than double quotes